src/components/Items.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import CheckBox from './CheckBox';
+import EditBox from './EditBox';
+import EditButton from './EditButton';
+import DeleteItem from './DeleteItem';
+
+export default function Items({items, setItems}) {
+
+    // Adding Items to the list.
+    const listItems = items.map((item) => {
+        return (
+            <ul className='list' key={item.id}>
```

Comment:
This is wrong. You need only one `<ul>` parent and multiple `<li>`. What is inside the `map` function will be repeated.

<img width="1716" alt="Screenshot 2023-03-15 at 17 16 46" src="https://user-images.githubusercontent.com/3438819/225373500-6a3e9bfb-05f5-404a-9d18-6dc781b89af4.png">